### ISSUE TITLE
Volunteer Task - Community Education Portal Design

### ISSUE THREAD

**Issue by emmairwin** [mozilla/participation-org#103]
# Community Education Portal Design

We need to bring the Education Portal into design standards.   Before we invest in LESS/CSS changes, it would be good to have a design mockup to work from.

http://education.mozilla-community.org/
## Prerequisites

None, but assume you have comfort creating mockups/wireframes and have talent  and personal interest in design.
## Participation Steps
1. Check out the [Education Portal](http://education.mozilla-community.org/), and the [Mozilla Style Guide](https://www.mozilla.org/en-US/styleguide/identity/mozilla/color/).
2. If you are interested in proposing a mockup, leave your name here so we avoid duplication of work (basically, if someone else is working on this, you can collaborate vrs  working alone :)
3. Submit your mockup here, if it's complete then it will be accepted and we will proceed with implementing it!

**Comment by yofiesetiawan** [comment]
Hi Emma, 

I'm interested to submit mockup, but it will take sometime to do it...

**Comment by emmairwin** [comment]
Thansk @yofiesetiawan   appreciate your time and interest!  Do you want to have a chat first?  Or just ping me when you have questions?

**Comment by yofiesetiawan** [comment]
Hi @emmairwin i would say, right now, we will need for logo design as well (first), which we expect Elio will help us on that. But while waiting for logo, i can start working on designing wireframe, will let you know when i already come up with result on this..

**Comment by yofiesetiawan** [comment]
Hi @emmairwin i've made a mockup for homepage, i consider to use smaller width for this site, since we have sort of little contents, so using wide kind of layout wouldn't be wise.

I've created a project on marvelapp, but i'll also share preview here...

![mce-01-home](https://cloud.githubusercontent.com/assets/2810823/9221596/ba73157a-4115-11e5-94a2-c1e10714ddc0.jpg)

https://marvelapp.com/5ef7j3

**Comment by emmairwin** [comment]
@yofiesetiawan this is AMAZING  , I will circulate it a bit for feedback but I have one question.  Since most of the content will be curriculum ie: http://education.mozilla-community.org/modules/participation_design_thinking/workshop_preparation/    Do you still think narrow fits, that width?  Or maybe when we do this, the side menu should be collapsible?

**Comment by yofiesetiawan** [comment]
Hi @emmairwin i'm proposing something like this for inside pages.

![mce-02-inside](https://cloud.githubusercontent.com/assets/2810823/9232398/a43f90ac-4157-11e5-8069-e699ee50d7b3.jpg)

**Comment by Mte90** [comment]
Little idea, add the social share button for improve the promotion?
The mockup is amazing :-D

**Comment by yofiesetiawan** [comment]
@Mte90 Thanks! there would be 2 kind of social share, to the content detail page, and to share the site main url? or just the share button for the site main url?

**Comment by Mte90** [comment]
I think that in the content detail page is better for share the focused content.

**Comment by yofiesetiawan** [comment]
This is an alternative for background, with contributors...

![mce-02-inside-bgv2](https://cloud.githubusercontent.com/assets/2810823/9232992/c7842aac-415a-11e5-885e-952955244403.jpg)

**Comment by emmairwin** [comment]
Inside page LOVE.  seriously this is so great.   I'm torn on the contributor background, on one side I like how it feels - but worry it would be too busy?  Others?

**Comment by yofiesetiawan** [comment]
Apply the new logo design from Elio... 

![mce-01-home-new-logo](https://cloud.githubusercontent.com/assets/2810823/9234098/02521e04-4161-11e5-9272-ec5440151a43.jpg)

**Comment by yofiesetiawan** [comment]
Adapting the new logo with increased width to 960px..

![mce-02-home](https://cloud.githubusercontent.com/assets/2810823/9234361/a219d002-4162-11e5-8a02-98f75501e385.jpg)

**Comment by yofiesetiawan** [comment]
Try to be consistent on footer (from Elio) i put dummy for footer yet..

![mce-02-home](https://cloud.githubusercontent.com/assets/2810823/9234534/920b2084-4163-11e5-8ad5-33a021ec85df.jpg)

**Comment by geroter** [comment]
Three pieces of feedback:

1) We should think about the purpose of the landing/home page and how content on this site will grow. My guess is that it needs content on the landing page. It also needs a more extensible menu of "curriculum" -- perhaps the teach.mozilla.org site design is a good example. Not sure if this is in scope of this project.

2) Please please the plain background. That other one is crazy busy.

3) I personally don't love the idea/proliferation of different logos for different things. Maybe this is a longer philosophical conversation, but my strong preference would be the Mozilla logo + a wordmark of "Community Education" (also need to think about that name!).

**Comment by yofiesetiawan** [comment]
Thanks @geroter 

1) maybe @emmairwin can help to answer on this one..
2) i think i also prefer the plain background..
3) i have no comment about this, it depends...

**Comment by emmairwin** [comment]
1) This site has been an means-to-an-end and never full scoped to our needs, having said that it is built on P2PU's course in a box which partly funded by Mozilla for Teach Like Mozilla. (as background)
For current purposes, I believe the front page is fine.  I'm looking a lot at how it's put together, understanding we can swap images, button names etc.  I prefer minimal where possible (for example badges should go).

2)  Agree on no background tiles.

 Having said that yes, the menu needs re-invented  to allow for up to 8 top level items, with 2-6 sub items.  Any ideas there?

on 3) I have to agree with George on Logo, I mean I like having it for branding within Mozilla (maybe that's something add on the footer of all curriculum), but that the site logo remain Mozilla which is recognizable as main content.

**Comment by emmairwin** [comment]
Lets put a hold on this until I have a chance to evaluate teach.mozilla.org (today or tomorrow), and then we can make decisions.  thanks!

**Comment by yofiesetiawan** [comment]
Don't worry, we are now still only digging ideas with layout. I mean, when we already clear about the site content, sitemap, and structure, it'll be much easier and faster to decide what kind of layout that can fit best for this website.

**Comment by elioqoshi** [comment]
I disagree that there is no need for a logo here. Education is a big enough functional area to have its own subbrand. Also, its not like we are  abandoning the Mozilla name in there. Just have a look at FSA, Reps, Mozilla Moderator, and Bug Sprints which work greatly like that.

**Comment by geroter** [comment]
I'm arguing the general and bigger point: I don't think it's good for our
aims as Mozilla to have subbrands.

A few reasons:
- They dilute our main brand in the world, the presence of which is key to
  fulfilling our mission.
- It risks creating silos.
- It's confusing for people new to Mozilla.

Can you help me understand why they are important and desirable?

**Comment by emmairwin** [comment]
Also are there examples of how other brands/especially in technology have handled this?

**Comment by geroter** [comment]
Let me ask the question another way: Do you have examples of other brands (particularly volunteer engaging organizations) that have the kind of mixed internal-external sub-brands that you see at Mozilla?

I don't have any examples, but that doesn't mean they don't exist :)

**Comment by emmairwin** [comment]
I did some testing on a fork of teach.mozilla.org which provided me with enough information to know it is a suitable substitute for the existing platform. Next steps before moving ahead with more mockups are:
- [ ] Make a decision! Stay with Jeykll (fine) or go with fork.  I understand how much work is involved both-ways, so can consult other stakeholders in making a decision. Server to host node.
- [ ] If we go with the teach.mozilla.org fork, consult with @ppapadeas on strategy for synchronizing changes to teach, but also pushing improvements we might make.  
- [ ] Create a sitemap as @yofiesetiawan has rightly suggested we do before mockup.  And from this decide on appropriate items for menu items.
- [ ] New mockup!
- [ ] Approve mockup and move on to development.

![full screen](https://cloud.githubusercontent.com/assets/60618/9286364/cfae35e0-42a6-11e5-8736-0ebec8118a47.png)
![reps](https://cloud.githubusercontent.com/assets/60618/9286368/db86faa0-42a6-11e5-8022-cd19fe74f43b.png)

**Comment by yofiesetiawan** [comment]
Hi, sorry, what are the things, that differentiate "teach.mozilla.org" to "education.mozilla-community.org"?

**Comment by emmairwin** [comment]
Hi @yofiesetiawan good question, here is a breakdown of the differences (cc/ @SOFTowaha this might help you too)

**As far as design**
- A fork of teach.mozilla.org codebase would require we focus on theme colors, and icons in a mockup. Layout would not change. 
- Staying with education.mozilla.org codebase means we have more flexibility in design.

**From a technical standpoint the differences are**
- teach.mozilla.org is a custom app being developed by the foundation, it comes with a roadmap, a product manager who have goals for participation in learning networks.  That roadmap includes embedding Discourse.
- education.mozilla.org is a (hacked) fork of P2PU's 'Course in a Box', which is built on Jekyll, which uses Github pages.

'**From a 'launching style changes' perspective:'**
- teach.mozilla.org  create a series of styles and icons to match mockup. Port over all content .
- education.mozilla.org create header, footer, main page tempaltes, new bootstrap styles.

**From a maintenance standpoint:**
- teach.mozilla.org stay in touch with teach development team, to merge in changes we can also leverage, and likewise make them aware of any improvements we make. Collaborative opportunities
- education.mozilla.org would be maintained by us, leveraging what we can from the Jekyll community (hard because we are quite a few commits behind).

**Comment by emmairwin** [comment]
Closing this one, as we start a new heartbeat next week.  Once the technical decision is made on w hich platform will open a new one.  Thanks for your patience and great work @yofiesetiawan

**Comment by elioqoshi** [comment]
@geroter I get your point, but then we are already too late. I for example believe that the Reps Polos have quite bad branding? Why? Because the only place you can read Mozilla is on the sleeve and on the neck. No one knows what ReMo is apart Mozillians.

**Comment by jgmac1106** [comment]
I would have to agree with @geroter and @eliqoshi as a volunteer contributor I have no idea what the Mozilla acronyms mean (MoFo, MoCo, ReMo, MoPa, etc). 

After awhile I started to infer what all these mean (if that is what you mean by internal brands) but when I got started it was somewhat intimidating. I needed a glossary.

**Comment by yofiesetiawan** [comment]
Sometimes, for new projects i like branding with something unique such as 'WebFWD' for example, that can just stand alone. Instead of something like Mozilla Community Education, Mozilla Webmaker (which now turns to an app), Mozilla Participation, Mozilla Learning Network, etc. In some cases, maybe it's proper, but maybe i prefer something new like code name 'Firefox', 'Thunderbird', etc. Just my personal thought. And we need to be careful when launching new project. We should research whether we can sync it to another existing project. And if we really launch new project, we need to be sure that it can be sustain. I'm wondering if we can just sync Mozilla Learning Networks, and Community Education to Mozilla Webmaker. Those are still about teaching the web...

**Comment by asdofindia** [comment]
@jgmac1106 does something like https://wiki.mozilla.org/Glossary help?

**Comment by jgmac1106** [comment]
@asdofindia Yes it does. Thanks.

**Comment by asdofindia** [comment]
@jgmac1106 No, I mean there's some problem in documentation (in other words, we aren't using the Wiki properly), if a new contributor can't figure out what something means.

**Comment by jgmac1106** [comment]
I think it was a symptom of a bigger problem that is being address across
participation. I have been contributing for four years (last two years
seriously) but everything I did was with webmaker.org and #teachtheweb. I
just filled out mozillian.org profile (like ten minutes ago) as I did not
know about it. Everything we did was through google groups (when I
started), webmaker.org, twitter, and eventually discourse.

For the last few years a lot of educators followed a different contribution
pathway that did not have us interact with resources (or people) that would
help navigate the acronym alphabet soup.

I feel like much of this has been addressed since the Portland all hands
last December. The emphasis on Git and the unifying of different
contribution communities under a single banner seems to progressing well.

The terminology serves a purpose of signifying membership. When you know
the specialized language of an affinity space like Mozilla you are
basically speaking a membership card. We just have to make sure tools like
the wiki are surfaced so language does not become a barrier.
